hw/ip/aes/model/aes_model.core

```diff
@@ -0,0 +1,26 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:model:aes:0.5"
+description: "AES Model"
+filesets:
+  files_cc:
+    files:
+      - crypto.c
+      - crypto.h: { is_include_file: true }
+      - aes.c
+      - aes.h: { is_include_file: true }
```

Comment:
change to

crypto.c: { file_type: systemVerilogSource}
crypto.h: { file_type: systemVerilogSource, is_include_file: true }
aes.c: { file_type: systemVerilogSource}
aes.h: { file_type: systemVerilogSource, is_include_file: true }

Reply:
Done

Reply:
edalize doesn't include files marked with cSource in the fileset passed to VCS. That needs to be sorted, using systemVerilogSource can do the trick as workaround in the meantime. Please comment appropriately. https://github.com/olofk/edalize/issues/70